### PR TITLE
fix: character rendering for svelecte

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description
Fix ZEN-322

A bit of a hot-fix, I don't know why special characters get rendered so oddly by svelecte.

### CODE REVIEW

**Comment by cabreraalex** [comment]
Yeah this is annoying it's hardcoded in the default formatter to escape HTML characters like < >

File: frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte [hunk 1]
```diff
@@ -61,6 +61,10 @@
 	function handleMoved(e: CustomEvent) {
 		elements = e.detail.items;
 	}
+
+	function optionsRenderer(o: { name: string }) {
+		return o.name;
+	}
```
Comment (Sparkier) [inline frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte:67]: This can't be an arrow function because of types I assume?

Comment (Sparkier) [reply frontend/src/routes/(app)/report/[owner]/[report]/+page.svelte:67]: Should we make this a util function as well? Used in many places. Don't know...

**Review by Sparkier — COMMENTED** [review]
Kinda bad that we have to do this... Guess there's no way around, though.

**Review by Sparkier — APPROVED** [review]

**Pull request merged**